themes/bootstrap3/js/lightbox.js

```diff
@@ -220,12 +204,18 @@ VuFind.register('lightbox', function Lightbox() {
             || obj.url.match(/MyResearch\/(?!Bulk|Delete|Recover)/)
           ) && flashMessages.length === 0
         ) {
-          var eventResult = _emit('VuFind.lightbox.login', {
-            originalUrl: _originalUrl,
-            formUrl: obj.url
-          });
+          let doRefresh = true;
+          const cancelRefresh = () => doRefresh = false;
+
+          VuFind.emit(
+            'lightbox.login',
+            obj.url,
```

Comment:
I note that the original emit function took the event name and an object containing options, whereas the new emit function takes a long list of arguments. I realize that with the new mechanism, things could theoretically work either way. I wonder if sticking with the "object containing arguments" approach instead of the long argument list might have some advantages, both in terms of greater code readability (since each argument can be given a meaningful label) and in terms of callback design (since if you want to utilize the last argument in the list, you don't have to build a lengthy argument list in your function to get to it).

I don't object to keeping things as-is if you think this is a better design -- just weighing options in case you hadn't already considered these things.